src/test/unit/extansion.test.ts

```diff
@@ -0,0 +1,46 @@
+import {
+  Env0EnvironmentsProvider,
+  Environment,
+} from "../../env0-environments-provider";
+import * as myExtension from "../../extension";
+import type vscode from "vscode";
+import { getCurrentBranchWithRetry } from "../../utils/git";
+jest.mock("../../utils/git");
+
+const mockGetCurrentBranch = (branchName: string) => {
+  const getCurrentBranchWithRetryPromise = Promise.resolve(branchName);
+  (getCurrentBranchWithRetry as jest.Mock).mockImplementation(
+    () => getCurrentBranchWithRetryPromise
+  );
+  return getCurrentBranchWithRetryPromise;
+};
+
+let environmentsDataProvider: Env0EnvironmentsProvider;
+let environmentsTree: vscode.TreeView<Environment>;
+
+describe("extension init", () => {
```

Comment:
wrap this in another `describe`, as `"init"` is only once "case" or "phase" your testing right now

Reply:
ef442ba8d343a5c076dc07853e10e7fe908da899